mobile-app/app/screens/WalletNavigator/screens/RestoreWallet/RestoreMnemonicWallet.tsx

```diff
@@ -25,18 +26,35 @@ export function RestoreMnemonicWallet (): JSX.Element {
       isDirty
     },
     getValues,
-    trigger
+    trigger,
+    setValue
   } = useForm({ mode: 'onChange' })
   const [recoveryWords] = useState<number[]>(Array.from(Array(24), (v, i) => i + 1))
   const [isSubmitting, setIsSubmitting] = useState(false)
   const [inputRefMap, setInputRefMap] = useState<Array<React.RefObject<TextInput>>>([])
   const { isLight } = useThemeContext()
 
+  function autoFillFromClipboard (): void {
+    void (async () => {
+      const text = await Clipboard.getStringAsync()
+      if (text === undefined) {
+        return
+      }
+      const words = text.replace(/[^A-Za-z ]+/g, '').split(' ')
+      recoveryWords.forEach((word) => {
+        setValue(`recover_word_${word}`, words[word])
+        trigger(`recover_word_${word}`)
+      })
+    })()
+  }
+
   useEffect(() => {
     recoveryWords.forEach((r) => {
       inputRefMap[r] = createRef<TextInput>()
       setInputRefMap(inputRefMap)
     })
+
+    // autoFillFromClipboard()
```

Comment:
Unused

Reply:
left for future use if feature is demanded more often